setup.cfg

```diff
@@ -27,7 +27,7 @@ packages = find:
 python_requires = >=3.7
 zip_safe = False
 install_requires =
-    celery>=5.1.0,<5.4
+    celery>=5.1.0,<5.5
```

Comment:
I guess the bump is in order to have support for the `broker_connection_retry_on_startup` flag?

Reply:
to be honest i do not remember correctly. normally those things are about removing the deprecation warnings. the flag exists already in earlier versions

i think i did that bump because of the `Fix DeprecationWarning: datetime.datetime.utcnow()` which is part of the 5.4 release